templates/data/opensearch/index.html

```diff
@@ -45,7 +45,7 @@ <h1>OpenSearch for the enterprise</h1>
            href="/contact-us"
            aria-controls="data-opensearch-modal">Contact us<span class="u-off-screen">about OpenSearch for the enterprise</span></a>
         <a class="p-button"
-           href="https://assets.ubuntu.com/v1/6dc3440a-Charmed%20OpenSearch%20-%20Datasheet%20-%20amended%2004%2006%202024.pdf"><span class="u-off-screen">Download the Charmed OpenSearch</span>Datasheet</a>
+           href="https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed%2520OpenSearch%2520-%2520Datasheet.pdf?version=0"><span class="u-off-screen">Download the Charmed OpenSearch</span>Datasheet</a>
```

Comment:
Not sure if there is some encoding or copy/paste issue, but this link in code doesn't work.

This seems to work: https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed%20OpenSearch%20-%20Datasheet.pdf?version=0

```suggestion
           href="https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed%20OpenSearch%20-%20Datasheet.pdf?version=0"><span class="u-off-screen">Download the Charmed OpenSearch</span>Datasheet</a>
```

Reply:
ooh i see. it's the copy/paste issue. i'll fix it now